internal/runners/clean/run_lin_mac.go

```diff
@@ -123,15 +136,11 @@ func removeInstall(cfg configurable) error {
 		aggErr = errs.Wrap(aggErr, "Failed to remove system files at %s: %v", appPath, err)
 	}
 
-	installPath, err := installation.InstallPathFromExecPath()
-	if err != nil {
-		aggErr = errs.Wrap(aggErr, "Could not get installation path")
-	}
-
-	if fileutils.DirExists(installPath) {
```

Comment:
Isn't your change leaving things like the .App around if you're only deleting the bin dir?

Reply:
This is restoring the uninstall functionality to what it was before. The branch folder will still exist with the `.state_install_root` file and the `state-installer` executable but the `bin` directory will be removed.

Reply:
Ok, that doesn't seem like a desirable outcome. We want the entire folder to be uninstalled, unless I am missing something?